Controllers/CardController.cs

```diff
@@ -1,25 +1,24 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Drawing;
-using System.Linq;
-using System.Security.Cryptography;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using CardDealer.Models;
 
 namespace CardDealer.Controllers
 {
+
+
+
     public class CardController : Controller
     {
         private readonly int minimumCards = 1;
         private readonly int maximumCards = 1000;
 
-        public IActionResult Index(int amount = 1, bool sorted = false, bool repeats = false)
+        public IActionResult Index(int amount = 100000, bool sorted = false, bool repeats = false)
```

Comment:
Change of default amount of cards will be reverted to 1. 100 thousand as a default is not desirable.